### ISSUE TITLE
Dashboard: Revisions tab - Using color to convey changes of two versions

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#114]
URL: https://hcdev.openplus.ca/en/node/60168/revisions/view/591938/597590/visual_inline
User Agent: Chrome
Language: English

Issue Description: When comparing two versions, there is green highlighted text that visually indicates the differences between two versions. This is not conveyed to some assistive technology users such as screen reader users.
![image](https://user-images.githubusercontent.com/88776404/137132788-6f07ccc2-857e-47e7-bb03-8cfb9d95a167.png)

Steps to reproduce:

1. Log into the Dashboard
2. Action on the "Build"
3. Select "Revisions" tab
4. To compare two revisions, use the radio buttons from column A and B and press the compare button
5. Green highlighted text appears

Expected results: The difference between two versions conveyed with color should be conveyed to all users including those who cannot see color can still perceive the information through another means. Something like "Revised text" should appear before the changed text.

**Comment by echouin** [comment]
@XiuJuanXia From OP: "this is related to the Diff module. There isn't much we can do about it unless you want us to rewrite the module.﻿"

**Comment by XiuJuanXia** [comment]
Issue still exists. See screenshot below:
![image](https://user-images.githubusercontent.com/88776404/157927901-1fb49bae-74c1-462c-bb3e-1c75c3bbb19a.png)

**Comment by echouin** [comment]
This is an issue that was reported on the Drupal Module. OP Reported the issue and provided a fix. From our point of view, this is resolved.

From OP: A report was made to the module maintainer. ﻿Furthermore, I am seeing the fix that I proposed in the link above in production right now:

![image](https://user-images.githubusercontent.com/88099454/157950285-0fb71620-4c17-44ea-9ff8-081b7b54b3cf.png)

**Comment by mmiron-HC** [comment]
@echouin As Xiu explained, this is still a failure. If this is out of your control, this will need to be identified as a product limitation until they fix it. If you agree, please tag this issue as Drupal limitation

**Comment by echouin** [comment]
@mmiron-HC we tagged it as "Reported" as the Node Fix was reported to Drupal.

**Comment by davidgotlieb** [comment]
OpenPlus has added code behind the scenes to identify highlighted text so that it can be identified by screen readers etc. While it still uses colour for sighted users, I believe colour blind individuals would still see it as shading, they just wouldn't be able to identify the colour itself.

**Comment by hydegeoff** [comment]
@echouin / @davidgotlieb 
The issue is fixed related to user with screen readers by saying "insertion start and insertion end when entering and leaving green background.  However, the light green rgb(204, 255, 204) on white rgb(255, 255, 255) background does not meet the WCAG non-text colour contrast.  It is currently 1.1:1 when it be should be 3:1.  Currently, the text colour is valid, but when choosing a darker green, you must ensure text colour 4.5:1.  See below: 
![Colour Contrast does not meet WCAG](https://user-images.githubusercontent.com/88147013/186256661-6ab591ff-e1b0-48f3-aa8e-0d5442edd6c0.jpg)

**Comment by hydegeoff** [comment]
After further investigation, this issue can be closed, because all text in green is also underlined so all visual users would be able understand.  A new issue #173 will raised for the contrast mentioned in the previous comment
